yaus/apps/admin/src/app/app.tsx

```diff
@@ -1,14 +1,52 @@
-import * as React from "react";
-import { Admin, Resource } from "react-admin";
-import jsonServerProvider from "ra-data-json-server";
-import { PostList } from "./posts";
-
-const App = () => (
-  <Admin
-    dataProvider={jsonServerProvider("https://jsonplaceholder.typicode.com")}
-  >
-    <Resource name="posts" list={PostList} options={{}} />
-  </Admin>
-);
-
-export default App;
\ No newline at end of file
+import * as React from 'react';
+import { Admin, Resource, DataProvider, ListGuesser } from 'react-admin';
+import buildHasuraProvider from 'ra-data-hasura';
+import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client';
+import { useState, useEffect } from 'react';
+import Dashboard from './components/Dashboard';
+
+import { createBrowserHistory as createHistory } from 'history';
+const history = createHistory();
+
+const createApolloClient = async () => {
+  console.log(process.env)
```

Comment:
Remove all console.logs. Also, add an eslint rule to check if you have any and let's make that part of the build.